files/postgres/docker-entrypoint-initdb.d/columnar.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-EOSQL
+  CREATE EXTENSION IF NOT EXISTS columnar;
+  ALTER EXTENSION columnar UPDATE;
+  ALTER DATABASE "${POSTGRES_DB}" SET default_table_access_method = 'columnar';
```

Comment:
Just discovered that the init script is only executed when the db is not created: https://github.com/docker-library/postgres/blob/cba2a05c03706daf5f9a66b93a447540b62df063/docker-entrypoint.sh#L312C13-L312C36. The above two lines are to update the columnar extension if it's already in use. They wouldn't be executed from what I can see.

Reply:
I wasn't aware of that, but I consider that mostly optimal behavior. It would be nice if we could run the `ALTER EXTENSION ... UPDATE` query every time, I guess there's not a built-in way for us to do that.